src/core/thread/mle_constants.hpp

```diff
@@ -57,22 +57,23 @@ enum
  */
 enum
 {
-    kVersion                       = 2,     ///< MLE Version
-    kUdpPort                       = 19788, ///< MLE UDP Port
-    kParentRequestRouterTimeout    = 750,   ///< Router Parent Request timeout
-    kParentRequestReedTimeout      = 1250,  ///< Router and REEDs Parent Request timeout
-    kAttachStartJitter             = 50,    ///< Maximum jitter time added to start of attach.
-    kAnnounceProcessTimeout        = 250,   ///< Timeout after receiving Announcement before channel/pan-id change
-    kAnnounceTimeout               = 1400,  ///< Total timeout used for sending Announcement messages
-    kMinAnnounceDelay              = 80,    ///< Minimum delay between Announcement messages
-    kParentResponseMaxDelayRouters = 500,   ///< Maximum delay for response for Parent Request sent to routers only
-    kParentResponseMaxDelayAll     = 1000,  ///< Maximum delay for response for Parent Request sent to all devices
-    kUnicastRetransmissionDelay    = 1000,  ///< Base delay before retransmitting an MLE unicast.
-    kMaxTransmissionCount          = 3,     ///< Maximum number of times an MLE message may be transmitted.
-    kMaxResponseDelay              = 1000,  ///< Maximum delay before responding to a multicast request
-    kMaxChildIdRequestTimeout      = 5000,  ///< Maximum delay for receiving a Child ID Request
-    kMaxChildUpdateResponseTimeout = 2000,  ///< Maximum delay for receiving a Child Update Response
-    kMaxLinkRequestTimeout         = 2000,  ///< Maximum delay for receiving a Link Accept
+    kVersion                        = 2,     ///< MLE Version
+    kUdpPort                        = 19788, ///< MLE UDP Port
+    kParentRequestRouterTimeout     = 750,   ///< Router Parent Request timeout
+    kParentRequestReedTimeout       = 1250,  ///< Router and REEDs Parent Request timeout
+    kAttachStartJitter              = 50,    ///< Maximum jitter time added to start of attach.
+    kAnnounceProcessTimeout         = 250,   ///< Timeout after receiving Announcement before channel/pan-id change
+    kAnnounceTimeout                = 1400,  ///< Total timeout used for sending Announcement messages
+    kMinAnnounceDelay               = 80,    ///< Minimum delay between Announcement messages
+    kParentResponseMaxDelayRouters  = 500,   ///< Maximum delay for response for Parent Request sent to routers only
+    kParentResponseMaxDelayAll      = 1000,  ///< Maximum delay for response for Parent Request sent to all devices
+    kUnicastRetransmissionDelay     = 1000,  ///< Base delay before retransmitting an MLE unicast.
+    kChildUpdateRequestPendingDelay = 1000,  ///< Delay (in ms) for aggregating Child Update Request.
```

Comment:
Do we need to wait 1 second?  Can this be smaller?

Reply:
shorted to 100ms in new patch. is it ok?